TransferMethod/Sources/SelectTransferMethodTypePresenter.swift

```diff
@@ -130,6 +130,7 @@ final class SelectTransferMethodTypePresenter {
                     .getKeys(completion: strongSelf.getKeysHandler(
                         success: { (result) in
                             guard let countries = result?.countries(), countries.isNotEmpty  else {
+                                view.hideLoading()
```

Comment:
why is hideLoading moved inside success and failure?

Reply:
When the Country and currency code api (1st GraphQL) is called first time, on success we call the transfer method types api (2nd GraphQL). So hide the progress for first call if it fails. Otherwise progress is hidden on receiving response of 2nd call.